wdn/templates_5.2/scss/critical.scss

```diff
@@ -1744,12 +1741,12 @@ a.unl-cream:link {
 // .unl-darkest-gray {
 //     color: #242423!important;
 // }
-// .unl-font-serif {
-//     font-family: Mercury SSm A,Mercury SSm B,Georgia,serif!important;
-// }
-.unl-font-sans {
-  font-family: Gotham SSm A,Gotham SSm B,Verdana,sans-serif!important;
+.unl-font-serif {
+  font-family: Mercury SSm A,Mercury SSm B,Georgia,serif!important;
 }
+// .unl-font-sans {
```

Comment:
Do you want it in commented out or just remove?

Reply:
The critical CSS was initially built by copying/pasting some common styles, commenting out all of them, then selectively uncommenting out the most important styles until we reached ~12–13kb. We could revisit the Sass file and clean it up, but if we do, let's do it in a separate PR.